.github/workflows/nix-auto-update.yaml

```diff
@@ -0,0 +1,29 @@
+name: update-flake-lock
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '0 0 * * 0' # runs weekly on Sunday at 00:00
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Install Nix
+        uses: cachix/install-nix-action@3715ab1a11cac9e991980d7b4a28d80c7ebdd8f9 # v27
+        with:
+          extra_nix_config: |
+            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
+      - name: Update flake.lock
+        uses: DeterminateSystems/update-flake-lock@a2bbe0274e3a0c4194390a1e445f734c597ebc37 # v24
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          git-author-name: 'Vincent Demeester (sbr-bot)'
+          git-author-email: 'bot@sbr.pm'
+          git-committer-name: 'Vincent Demeester (sbr-bot)'
+          git-committer-email: 'bot@sbr.pm'
```

Comment:
@chmouel what do you want me to put here ?

Reply:
whatever it's good this